src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.scss

```diff
@@ -1,3 +1,12 @@
+button {
```

Comment:
To keep things consistent with how the buttons are styled in the sessions table, you can take a look at the `actions-cell` class in this component `src/web/app/components/sessions-table/sessions-table.component.html`.

Reply:
Yup I've seen the scss file in that component, and they only used `margin`. But in the current courses page, there's already `margin-right` for class `.btn-left`, that's why I just added the top and bottom margin. Unless you meant to remove the `.btn-left` styling and combine it all together in `.button`, I can do that!

Reply:
Yes that would be ideal, since `.btn-left` is manually created anyway.